ui/src/components/Navbar/Navbar.scss

```diff
@@ -109,7 +113,7 @@
 }
 
 //small screen
-@media screen and (max-width: $aleph-screen-xs-max-width) {
+@media screen and (max-width: $aleph-screen-sm-max-width) {
```

Comment:
In general: I think it's much more readable if we keep the media queries close to where the objects are declared, not as a separate secondary structure. Instead of: 

```
.Navbar {
    .h2 { 
       color: red;
    }
}

@media (...) {
   .Navbar { 
        h2 { color: blue; }
   }
}
```

It's much cleaner to do: 

```
.Navbar {
  h2 {
     color: red;
     @media () {
          color: blue
       }
    }
}
```

See also: http://thesassway.com/intermediate/responsive-web-design-in-sass-using-media-queries-in-sass-32

Reply:
@pudo i get what you are saying, i saw these kind of media queries and how they write it, but in one file (for example Navbar.scss) i have to change more than 15 classes, it will get really messy if i add for all of the classes media query and there will be a lot more lines of code then when you put it in one place. But I can do that in smaller scss files, but then it would not be consistent.